### ISSUE TITLE
translation for Chinese

### ISSUE THREAD

**Issue by bubifengyun** [kartik-v/yii2-tree-manager#96]
https://github.com/kartik-v/yii2-tree-manager/blob/master/messages/zh-CN/kvtree.php

I am Chinese, it may not be translated very well, 
can be follows

```php
<?php
/**
 * Message translations.
 *
 * This file is automatically generated by 'yii message/extract' command.
 * It contains the localizable messages extracted from source code.
 * You may modify this file by translating the extracted messages.
 *
 * Each array element represents the translation (value) of a message (key).
 * If the value is empty, the message is considered as not translated.
 * Messages that no longer need translation will have their translations
 * enclosed between a pair of '@@' marks.
 *
 * Message string can be used with plural forms format. Check i18n section
 * of the guide for details.
 *
 * NOTE: this file must be saved in UTF-8 encoding.
 */
return [
    '(new)' => '(新建)',
    'Active' => '已激活',
    'Add new' => '添加节点',
    'Add new root' => '添加根节点',
    'Admin Settings' => '管理员设置',
    'Already at bottom-most node in the hierarchy.' => '已经在结构最底节点',
    'Already at left-most node in the hierarchy.' => '已经在结构最左节点',
    'Already at right-most node in the hierarchy.' => '已经在结构最右节点',
    'Already at top-most node in the hierarchy.' => '已经在结构最顶节点',
    'Are you sure you want to remove this node?' => '确定删除该节点？',
    'Cannot create node. Parent node is not saved or is invalid.' => '无法创建节点，父节点未保存或无效。',
    'Cannot move this node as the node details are not saved yet.' => '节点内容尚未保存，无法移动该节点。',
    'Clear search results' => '清空搜索结果',
    'Collapsed' => '关闭',
    'Default' => '默认',
    'Delete' => '删除',
    'Depth' => '层级',
    'Disabled' => '不可操作',
    'Error removing the node. Please try again later.' => '删除节点出错，请稍后重试。',
    'Error while creating the node. Please try again later.' => '创建节点出错，请稍后重试。',
    'Error while moving the node. Please try again later.' => '移动节点出错，请稍后重试。',
    'Error while removing the node. Please try again later.' => '删除节点出错，请稍后重试。',
    'Error while saving the node. Please try again later.' => '保存节点出错，请稍后重试。',
    'Error while viewing the node. Please try again later.' => '查看节点出错，请稍后重试。',
    'ID' => 'ID编号',
    'Icon' => '图标',
    'Icon Type' => '图标类型',
    'Left' => '左',
    'Movable Down' => '可以下移',
    'Movable Left' => '可以左移',
    'Movable Right' => '可以右移',
    'Movable Up' => '可以上移',
    'Move Down' => '下移',
    'Move Left' => '左移',
    'Move Right' => '右移',
    'Move Up' => '上移',
    'Name' => '名称',
    'No valid tree nodes are available for display. Use toolbar buttons to add tree nodes.' => '没有有效的节点可以显示，请用工具条里按钮添加节点。',
    'Node # {id} - \'{name}\': {error}' => '节点 # {id} - \'{name}\': {error}',
    'Read Only' => '只读',
    'Refresh' => '刷新',
    'Removable' => '可删除',
    'Removable (with children)' => '可删除（附带子节点）',
    'Reset' => '重置',
    'Right' => '右',
    'Root' => '根',
    'Save' => '保存',
    'Saved the node details successfully.' => '节点保存成功。',
    'Search...' => '搜索...',
    'Select a node by clicking on one of the tree items.' => '点击树形结构某一项可以选中该节点。',
    'Select...' => '选择...',
    'Selected' => '选中',
    'The node was moved successfully.' => '移动节点成功。',
    'The node was removed successfully.' => '删除节点成功。',
    'The node was successfully created.' => '创建节点成功。',
    'The selected node cannot be moved.' => '选中的节点不可被删除。',
    'The untitled node was removed.' => '无标题的节点被成功删除。',
    'This operation is not allowed.' => '无权操作。',
    'Untitled' => '无标题',
    'Visible' => '可见',
];
```

**Comment by bubifengyun** [comment]
Sorry, the first time nothing code was added in the above.

**Comment by kartik-v** [comment]
You can submit it as a pull request and I can merge it.

**Comment by bubifengyun** [comment]
thank you.

you can copy it instead of original file. 
but,
submit request and merge cost much time and energy.

**Comment by kartik-v** [comment]
No actually this will manually involve more effort copy pasting into local and then again pushing code to this repo.

Managing and merging changes via Pull Request is the recommended way to track changes by different users to this repo.

**Comment by kartik-v** [comment]
Closed via #99.
